addons/mail/static/src/components/composer_suggested_partner/composer_suggested_partner.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ComposerSuggestedPartner" owl="1">
+        <div class="o_ComposerSuggestedPartner" t-attf-title="Add as recipient and follower (reason: {{suggestedPartner.reason}})">
+            <div class="custom-control custom-checkbox">
+                <input t-on-change="_onChangeSuggestedPartner" type="checkbox" class="custom-control-input" t-attf-id="o_composer_suggested_partners_p{{suggestedPartner.localId}}" t-att-checked="suggestedPartner.checked ? 'checked' : undefined" />
```

Comment:
Let component generate a unique id itself with `_.uniqueId()` and make use of this value as id of this part in template, using our naming convention `o_ComposerSuggestedRecipient_{{ componentUniqueId }}_checkbox` (btw 1st time using id, that's why suggestion to put id between `ComponentName` and `part`).